packages/components/.storybook/preview-head.html

```diff
@@ -44,4 +44,9 @@
     .virtualized-list div.tc-list-cell-modified {
         flex: 0 0 90px;
     }
+
```

Comment:
can make sense to show how to switch to different number of column, but as its not done using props i guess this shouldn't be here.

Reply:
Ok I removed it